.github/scripts/publish_post_check.sh

```diff
@@ -0,0 +1,105 @@
+
```

Comment:
Please remove any changes to Actions workflow configs.

Reply:
Done. (Must've been a bad merge)